test/test_framework.py

```diff
@@ -1109,6 +1115,9 @@ def _test_buildvocab(tmpdir, run_num, multi=False):
         }
 
     os.environ["DATA_DIR"] = os.path.join(_test_dir(), "corpus")
+    os.environ["ALIGN_DIR"] = os.path.join(
```

Comment:
The `DATA_DIR` variable is deleted at the end of the test. Let's do the same with `ALIGN_DIR` to make sure it does not unexpectedly impact other tests.